_data/fr/navigation.yml

```diff
@@ -0,0 +1,28 @@
+nav:
+- title:                 Utilisation
+  subfolderitems:
+    - page:              Démarrage et installation
```

Comment:
"Premier pas" could be a less descriptive translation, fitting best with the English version

Reply:
@Saimyx a utilisé « Pour commencer » dans son travail, je trouve cette formulation plus approprié et parlante que « Premier pas ».
Mais c'est un détail.

Reply:
(keep it in English)

"Pour commencer" is fine too.